src/main/java/com/datatheorem/mobileappsecurity/jenkins/plugin/SendBuildAction.java

```diff
@@ -120,7 +120,35 @@
         this.proxyUnsecureConnection = proxyUnsecureConnection;
     }
 
-    public SendBuildMessage perform(String buildPath, String sourceMapPath, Boolean isBuildStoredInArtifactFolder) {
+    public SendBuildMessage perform(
+            String buildPath,
+            String sourceMapPath,
+            Boolean isBuildStoredInArtifactFolder
+    ) {
+        /*
+         * Perform the SendBuildAction : send the build to Data Theorem Upload API
+         * @param :
+         *    buildPath : Path of the build we want to send to Data Theorem
+         * @return :
+         *    SendBuildMessage containing the success or the failure information about the SendBuild process
+         */
+        SendBuildMessage uploadMessage = new SendBuildMessage(false, "");
+        for(int retry=0; retry<3; retry++){
```

Comment:
I think the Java conventions for white spaces is that every binary operators except `.` should have both operands separated by a space, and there should be a space between the `for` keyword and the opening parenthesis, as well as between the closing parenthesis and the opening bracket.